<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSS Demo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1a1a1a;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="bgFrame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>700</width>
      <height>500</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="top_frame">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="title_lb">
          <property name="text">
           <string>QSS Demo</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="mini_btn">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="close_btn">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="left_frame">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="avatar_btn">
            <property name="text">
             <string>Avatar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="home_btn">
            <property name="text">
             <string>Home</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="personal_btn">
            <property name="text">
             <string>Personal</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="report_btn">
            <property name="text">
             <string>Report</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="notes_btn">
            <property name="text">
             <string>Notes</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pay_btn">
            <property name="text">
             <string>Pay</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dark_light_btn">
            <property name="text">
             <string>Dark/Light</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_home">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Home Page</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLineEdit" name="username_edit">
               <property name="placeholderText">
                <string>Username</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="password_edit">
               <property name="placeholderText">
                <string>Password</string>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="checkBox">
               <property name="text">
                <string>Remember me</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="login_btn">
               <property name="text">
                <string>Login</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="root_login_btn">
               <property name="text">
                <string>Root Login</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="register_btn">
               <property name="text">
                <string>Register</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QFrame" name="notice_frame">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Notifications</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="notice_list">
                <item>
                 <property name="text">
                  <string>Notification 1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Notification 2</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Notification 3</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_personal">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Personal Information</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="changeAvatar_btn">
             <property name="text">
              <string>Change Avatar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="userName_edit">
             <property name="placeholderText">
              <string>Username</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="realname_edit">
             <property name="placeholderText">
              <string>Real Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="phone_edit">
             <property name="placeholderText">
              <string>Phone Number</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>Option 1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Option 2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Option 3</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton">
             <property name="text">
              <string>Option A</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_2">
             <property name="text">
              <string>Option B</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveChange_btn">
             <property name="text">
              <string>Save Changes</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
 </connections>
</ui>